using-binary-decoding.py

```diff
@@ -0,0 +1,4 @@
+import codecs
+hex = [b"48", b"65", b"6c", b"6c", b"6f", b"20", b"57", b"6f", b"6c", b"64", b"0a"]
```

Comment:
Missing the letter "r" as well as the "!" at the end before the line feed character.

I suggest changing the hex array to

```py
hex = [b"48", b"65", b"6c", b"6c", b"6f",
       b"20", b"57", b"6f", b"72", b"6c", b"64", b"21", b"0a"]
```

should do the trick. I believe everything else is in order.